### DISCUSSION TITLE
Health Check now 97.8% V 24.5

### DISCUSSION THREAD

**Discussion by ronrosenfeld** [nzbgetcom/nzbget#466] [Q&A] [answered]
I recently upgraded to V24.5 (Docker on Synology). I tried to download three episodes of a new show but they all failed with `...due to health 97.7% below critical 97.8%`. My recollection is that previous versions had the critical level set at around 89-90%.  What was the reason for this change? I did not see any mention of this in the Release notes.

**Comment by dnzbk** [comment]
There wasn't such change. At least not intentionally. 
Could you share the NZB for testing? 
denis@nzbget.com

**Reply by ronrosenfeld** [reply]
The reason I thought there was such a change is because of the `warning` log entry below at `Thu Dec 12 2024 13:50:29`
```
Kind | Time | Text
-- | -- | --
info | Thu Dec 12 2024 13:50:32 | Collection - added to history
info | Thu Dec 12 2024 13:50:32 | Post-process-script NotifySickChill for - successful
info | Thu Dec 12 2024 13:50:32 | Notify SickChill: Post-processing started successfully: Started post-process for /downloads/intermediate/-.#7442
info | Thu Dec 12 2024 13:50:31 | Executing post-process-script NotifySickChill for -E
info | Thu Dec 12 2024 13:50:31 | Post-process-script FakeDetector for - skipped
info | Thu Dec 12 2024 13:50:31 | Fake Detector: Destination directory doesn't exist, exiting
info | Thu Dec 12 2024 13:50:31 | Executing post-process-script FakeDetector for -
info | Thu Dec 12 2024 13:50:30 | Queueing - for post-processing
info | Thu Dec 12 2024 13:50:30 | Collection - deleted from queue
detail | Thu Dec 12 2024 13:50:29 | Deleting file -.vol-03.par2 from download queue
detail | Thu Dec 12 2024 13:50:29 | Deleting file -.vol-05.par2 from download queue
detail | Thu Dec 12 2024 13:50:29 | Deleting file -vol-06.par2 from download queue
detail | Thu Dec 12 2024 13:50:29 | Deleting file -.vol-04.par2 from download queue
detail | Thu Dec 12 2024 13:50:29 | Deleting file -.vol-02.par2 from download queue
detail | Thu Dec 12 2024 13:50:29 | Deleting file -.vol-01.par2 from download queue
detail | Thu Dec 12 2024 13:50:29 | Deleting file -.vol-07.par2 from download queue
detail | Thu Dec 12 2024 13:50:29 | Deleting file caYN9ATO9fT2uA4XxY6JHM9q7seQPGYQ.mkv from download queue
warning | Thu Dec 12 2024 13:50:29 | Cancelling download and deleting - due to health 97.5% below critical 97.6%
info | Thu Dec 12 2024 13:50:21 | Fake Detector: Skipping sorting since could not find any rar-files
info | Thu Dec 12 2024 13:50:21 | Fake Detector: Sorting inner files for earlier fake detection for -
```

Attached is the nzb file renamed to a .txt file so it would attach
[deleted]

I don't know if the item would have downloaded anyway, but I was concerned about the critical health percent mentioned in the warning message.

Thanks

**Comment by dnzbk** [comment]
Thank you.

Tested and everything looks good except for the bad deobfuscation: `caYN9ATO9fT2uA4XxY6JHM9q7seQPGYQ.mkv` - that's what I'm currently working on. 
I'm not sure what caused such warning on your side. Could you try to redownload it? 
What usenet provider? I'm using eweka. 

Please send such information to my e-mail next time. Had to moderate your comment, sorry.

**Comment by ronrosenfeld** [comment]
Hi Denis,

I had tried downloading several times, and several different NZB’s of both that episode and others of the first three episodes of that particular show.
All returned the same error message.

During the same time frame, I was able to download other shows without any issues.

My primary usenet provider is Supernews, and my backup provider is Astraweb.
And I just checked to confirm that my accounts are current on both sites.

What does “bad deobfuscation” mean in the setting of my problems with downloading these episodes?

Thanks.
    Ron

**Comment by dnzbk** [comment]
Confirmed.

My primary provider is Eweka, and I use Astraweb as a backup. If I use only Astraweb it fails with exactly the same error message.
But Eweka works perfectly. No errors. 
Apparently the problem is ISP related and not the app.

By “bad deobfuscation” I meant an output file with a meaningless name such as “caYN9ATO9fT2uA4XxY6JHM9q7seQPGYQ.mkv”. 
It has nothing to do with your problem. I just mentioned it in case of such questions. Sorry for the confusion.

**Reply by ronrosenfeld** [reply]
Good to hear. And I guess I will have to mitigate that issue by using a different provider for particular shows. 

However, what about that error message implying that the critical health level is 97.6%?  Shouldn't critical health be much lower?

**Reply by dnzbk** [reply]
I think it's ok because NZBGet can't even download par2 files to do verification. 
If you switch CHECK AND REPAIR->HealthCheck to None, you get 100% failed articles, I guess.

**Comment by ronrosenfeld** [comment, marked as answer]
Hi Denis,

After discussion with Supernews support, I discovered that the problem is that the articles had been pulled due to copyright issues. (I suppose the same thing happened at Astraweb).

In your experience, has this ever been an issue with Eweka? If not, I'll probably switch to them for primary.

Also, with regard to the Critical Health percentage issue, by clicking on an item while it is downloading, I can see the current health and critical health. The critical health number seems to vary. With the failed item, it was 97%; but with a valid item it was 94% at some point in the process. I would have thought with that type of name it would be a constant, but maybe it is a variable depending on other factors.

Ron
________________________________

**Reply by luckedea** [reply]
@ronrosenfeld 

Usually critical health is around 90% if par files are fine and file can be restored. If critical health goes up, that's usually rare, means there's something wrong with par files and content cannot be restored.

As for the completion per news servers - this map from wikipedia might be useful https://en.wikipedia.org/wiki/Usenet#/media/File:Usenet_Providers_and_Backbones.svg

**Reply by ronrosenfeld** [reply]
Thanks for that information. Conceptually I would have thought the Critical Health value would be a constant, and the actual health a variable. Suprising (to me) that both can vary. Clearly I'm just a novice at this conceptualizing and not understanding how Critical Health is determined. But the bottom line, for me, is that I understand why the downloads failed (DMCA) and thank you for pointing me to Eweka, which I guess uses NTB and DMCA and, at least, had the articles I was looking for.
